src/core/PlayHandle.cpp

```diff
@@ -22,18 +22,23 @@
  *
  */
  
-#include "PlayHandle.h"
 #include "BufferManager.h"
+#include "Engine.h"
+#include "Mixer.h"
+#include "PlayHandle.h"
```

Comment:
Why this line have been moved from above? It should stay above `#include "BufferManager.h"` according to our convention, and that way seems more consistent for me.

Reply:
No reason, probably an unintentional change.